src/posts/2020-10-14-wcag2.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Introduction to WCAG 2
+description: WCAG is a set of accessibility guidelines published by WAI...
```

Comment:
Can we have a more descriptive description? It should, in 1-2 sentences, inform the reader about what they can expect to read about in the post.

Reply:
Yes we can. What do you think about  " A quick overview of WCAG 2 documents" ?